docker/download-mutinynet.sh

```diff
@@ -3,15 +3,37 @@
 # This file downloads the mutinynet docker-compose files for the LN gateway and fedimintd
 # You can download and run it with: curl -sSL https://raw.githubusercontent.com/fedimint/fedimint/master/docker/download-mutinynet.sh | bash
 
+if ! [ -x "$(command -v docker-compose)" ]; then
+  # check if we are running as root
+  if [ "$EUID" -ne 0 ]; then
+    echo 'Error: docker-compose is not installed and we can not install it for you.' >&2
+    exit 1
+  fi
```

Comment:
Should we encourage running our scripts a root? I'd much rather just print the command that is likely to work on the machine but leave it to the user to execute it as root.

Reply:
Good point. But I don't think it's a big encouragement. It isn't like we are saying "please try again running this script as root so it can install the dependencies"